### PR TITLE
tests: update documentation and checks for interfaces tests

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This change adds extra self documentation and improbe organization and checks for interfaces tests.

### CODE REVIEW

File: tests/main/interfaces-daemon-notify/task.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ execute: |
         fi
         sleep 1
     done
-    [ "$denials_before" -eq "$denials_after" ]
+    test "$denials_before" != "$denials_after"
```
Comment (mvo5) [inline tests/main/interfaces-daemon-notify/task.yaml:35]: This looks incorrect, the `-eq` should translate to `=` here - or am I missing something?

**Review by mvo5 — COMMENTED** [review]
Looks good, thanks for doing these cleanups! Just one question inline, then it's good to go :)

**Review by bboozzoo — APPROVED** [review]

File: tests/main/interfaces-daemon-notify/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/main/interfaces-daemon-notify/task.yaml:35]: nice catch, thanks for the review

File: tests/main/interfaces-hooks/task.yaml [hunk 2]
```diff
@@ -65,36 +64,36 @@ execute: |
 
     remove_markers
 
-    # make sure disconnect hooks are executed
+    echo "Make sure that when the interface is disconnected hooks are executed"
```
[the remark this answers is not in the document]
Comment (sergiocazzolato) [reply tests/main/interfaces-hooks/task.yaml:67]: Nice, thanks for the review

**Review by mvo5 — APPROVED** [review]

**Pull request merged**